content/docs/digging_deeper/transmit.md

```diff
@@ -0,0 +1,373 @@
+# Transmit
+
+Transmit is a native Server-Sent-Event (SSE) module built for AdonisJS. It is a simple and efficient way to send real-time updates to the client, such as notifications, live chat messages, or any other type of real-time data.
+
+:::note
+The data transmission occurs only from server to client, not the other way around. 
+:::
+
+## Installation
+
+Install and configure the package using the following command :
+
+```sh
+node ace add @adonisjs/transmit
+```
+
+:::disclosure{title="See steps performed by the add command"}
+
+1. Installs the `@adonisjs/transmit` package using the detected package manager.
+ 
+2. Registers the `@adonisjs/transmit/transmit_provider` service provider inside the `adonisrc.ts` file.
+ 
+3. Creates a new `transmit.ts` file inside the `config` directory.
+ 
+:::
+
+## Configuration
+
+The configuration for the transmit package is stored within the `config/transmit.ts` file.
+
+See also: [Config stub](https://github.com/adonisjs/transmit/blob/main/stubs/config/transmit.stub)
+
+```ts
+import { defineConfig } from '@adonisjs/transmit'
+
+export default defineConfig({
+  pingInterval: false,
+  transport: null
+})
+```
+
+<dl>
+
+<dt>
+
+pingInterval
+
+</dt>
+
+<dd>
+
+The interval used to send ping messages to the client. The value is in milliseconds or using a string `Duration` format (i.e: `10s`). Set to `false` to disable ping messages.
+
+</dd>
+
+<dt>
+
+transport
+
+</dt>
+
+<dd>
+
+Transmit supports syncing events across multiple servers or instances. You can enable the feature by referencing the wanted transport layer (only `redis` is supported for now). Set to `null` to disable syncing.
+
+```ts
+import env from '#start/env'
+import { defineConfig } from '@adonisjs/transmit'
+import { redis } from '@adonisjs/transmit/transports'
+
+export default defineConfig({
+  transport: {
+    driver: redis({
+      host: env.get('REDIS_HOST'),
+      port: env.get('REDIS_PORT'),
+      password: env.get('REDIS_PASSWORD'),
+    })
+  }
+})
+```
+
+:::note
+Ensure you have `ioredis` installed when using the `redis` transport.
+:::
+
+</dd>
+
+</dl>
+
+## Channels
+
+Channels are used to group events. For example, you can have a channel for notifications, another for chat messages, and so on.
+
+### Channel Names
+
+Channel names are used to identify the channel. They are case-sensitive and must be a string. You cannot use any special characters or spaces in the channel name except `/`. The following are some examples of valid channel names:
+
+```ts
+transmit.broadcast('global', { message: 'Hello' })
+transmit.broadcast('chats/1/messages', { message: 'Hello' })
+transmit.broadcast('users/1', { message: 'Hello' })
+```
+
+:::tip
+Channel names use the same syntax as route in AdonisJS.
```

Comment:
But they are not related to routes and this is by design since it's just a key. 

I think it worth mentioning it to avoid issue where people think that they could not use a name because it's already used in a route.